components/st-microelectronics/stm-api.stanza

```diff
@@ -142,25 +143,34 @@ public defn set-boot (stm:JITXObject, boot-from:String) :
     if has-pin?(stm, "BOOT[1]") : 
       check connected(stm.BOOT[1])
 
-defn supports? (module:InstantiableType, bundle:PortType) -> True|False :
+defn supports? (module:InstantiableType, bundle:Bundle) -> True|False :
   pcb-module temp :
     inst i : module
     require r:bundle from i
   set-main-module(temp) ;uh oh
   val result = assign-pins?()
   result is-not False
 
-public defn connect-debug (stm:JITXObject, debug-interface:PortType, connector:Instantiable) :
+public defn connect-debug (stm:JITXObject, debug-interface:Bundle, connector:Instantiable) :
   inside pcb-module:
     if supports?(instantiable-type(stm), debug-interface) and supports?(connector, debug-interface):
-      inst debug-con : connector
+      public inst debug-con : connector
```

Comment:
It's possible for the `if ...` statements not to resolve and connections will not be made. Making the connector `public` allows a user to remedy this within their design without crashing.